001/main.go

```diff
@@ -3,11 +3,8 @@
 // - Run the program
 // - Make it an executable
 
-
 package main
 
-import "fmt"
```

Comment:
why Delete the import Formatted Library? @raghavgh

Reply:
@blueprismo there is no usage of this import in this file then why we are importing it

Reply:
Because it's an exercise pending to do... not a solution.
I appretiate that you want to contribute, but read the first 2 line of this file or the README.md

Reply:
ohh ok sorry @blueprismo i misinterpreted the repo let me re add it.

Reply:
Nice :) Now i can accept it, happy hacktoberfest ;) @raghavgh ++